### DISCUSSION TITLE
User report

### DISCUSSION THREAD

**Discussion by fftxjg** [opendcim/openDCIM#1501] [Q&A]
Is there plan to create report which will summarize DCIM application users and they privilege? Something for access and rights review.
I see only report which is showing users in department. 
Thank you for sharing what is possible.

**Comment by samilliken** [comment]
Nope

 

From: fftxjg 
Sent: Friday, January 26, 2024 1:58 PM
To: opendcim/openDCIM ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [opendcim/openDCIM] User report (Discussion #1501)

 

Is there plan to create report which will summarize DCIM application users and they privilege? Something for access and rights review.
I see only report which is showing users in department.
Thank you for sharing what is possible.

**Comment by alex001x** [comment]
Ok

**Comment by samilliken** [comment]
No, no, no.   People without a clue need to stay out of the database, even with read statements.

 

Use the API.

 

Scott
